src/pythermalcomfort/jos3_functions/construction.py

```diff
@@ -212,15 +280,15 @@ def bfb_rate(
     else:  # age >= 70
         ci *= 0.7
 
-    bfb_all = ci * bsa_rate(height, weight, bsa_equation) * _BSAst.sum()  # [L/h]
+    bfb_all = ci * bsa_rate(height, weight, bsa_equation) * Default.local_bsa.sum()  # [L/h]
     return bfb_all / 290
```

Comment:
Why are we dividing this term by 290? Could you please add a comment in which we explain what 290 is?

Reply:
Thank you for pointing this out. This is really wrong way to code. 290 is the blood flow rate in the whole body. I added this to the Default class and used it in the function instead of using a numerical value.